package.json

```diff
@@ -14,9 +14,11 @@
     "node": "^12"
   },
   "scripts": {
-    "clean": "rm -rf src/*/node_modules/ build/ node_modules/ src/server_manager/install_scripts/do_install_script.ts",
+    "clean": "rm -rf src/*/node_modules/ build/ node_modules/ src/server_manager/install_scripts/do_install_script.ts third_party/shellcheck/download/",
     "do": "bash ./scripts/do_action.sh",
-    "lint": "tslint 'src/**/*.ts' -e '**/node_modules/**'"
+    "lint": "yarn shellcheck && yarn tslint",
+    "shellcheck": "find src scripts third_party/shellcheck -name *.sh -exec third_party/shellcheck/run.sh {} +",
```

Comment:
When I run on cygwin this finds errors in src/server_manager/bower_components/intl-messageformat/tests/saucelabs.sh.  Maybe we can ignore src/server_manager/bower_components?

Reply:
OK, I changed this to ignore `bower_components`.  Could you try again?

This made the `find` invocation too complex to be in a string in `package.json`, so I moved it into `scripts/shellcheck.sh`.

Reply:
@JonathanDCohen I think you have an unclean client. We no longer use bower.